validation/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -1,256 +1,38 @@
-= Validation Module - Mule 4
+= Validation Module 1.4 - Mule 4
 :page-aliases: connectors::validation/validation-connector.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-Validation Module v1.4
+The Validation module verifies that the content of a message in a Mule flow matches specific criteria. The module provides explicit messages about the cause of an exception in a flow. Based on the specific data you are validating, you can customize the exception message that is displayed in the logs.
```

Comment:
This paragraph is not clear to me: we start by saying that the module VERIFIES content against criteria, but then follow that by saying the the module PROVIDES MESSAGES about flow exceptions. Which action is the purpose of the module? These sound like different things to me.

The third sentence kind of clarifies this: does this mean that the module 1) verifies content against criteria (that you set) AND THEN 2) displays a message (that you configure) if the content does not match the criteria?

Clarify.